<launch>

    <arg name="static" default="3ne/map_static.txt" />
    <arg name="sunny" default="3ne/map_sunny.txt" />
    <arg name="coord" default="3ne/coordinates_real.yaml" />

    <param name="path_static" value="$(arg static)" />
    <param name="path_sunny" value="$(arg sunny)" />
    <param name="path_coord" value="$(arg coord)" />

    <node name="bwi_nav_reasoning_test" pkg="bwi_nav_reasoning" type="bwi_nav_reasoning_test" output="screen" />

</launch>
